src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ## Unreleased
 
-* `ActivityProcessor` implements IDisposable
+* `ActivityProcessor` implements IDisposable.
+  * When `Dispose` occurs, it calls `ShutdownAsync`.
```

Comment:
Need to highlight that user should override their Dispose method, if they want to achieve custom behavior.

Reply:
Just updated